src/Pyz/Glue/DocumentationGeneratorOpenApi/DocumentationGeneratorOpenApiConfig.php

```diff
@@ -1,47 +0,0 @@
-<?php
-
-/**
- * This file is part of the Spryker Suite.
- * For full license information, please view the LICENSE file that was distributed with this source code.
- */
-
-namespace Pyz\Glue\DocumentationGeneratorOpenApi;
-
-use Spryker\Glue\DocumentationGeneratorOpenApi\DocumentationGeneratorOpenApiConfig as SprykerDocumentationGeneratorOpenApiConfig;
-
-class DocumentationGeneratorOpenApiConfig extends SprykerDocumentationGeneratorOpenApiConfig
```

Comment:
Please make sure nonsplit has a project only annotation on this, so this is not missed on future publishs.